src/blob/index/bptree/tests.rs

```diff
@@ -154,7 +154,7 @@ async fn magic_byte_corruption() {
 
     let findex = BPTreeFileIndex::<KeyType>::from_file(
         FileName::from_path(&Path::new(filename)).expect("failed to create filename"),
-        None,
+        IODriver::default(),
```

Comment:
`IODriver` creates multiple times in this function. I suggest to do it only once at the beginning